src/plugins/intel_gpu/tests/test_cases/arg_max_gpu_test.cpp

```diff
@@ -1013,3 +1014,2050 @@ TEST(top_k_layer_tests, sort_probabilities_by_indices) {
         EXPECT_EQ(out_buffer[i], ref_vec[i]);
     }
 }
+
+const std::vector<float> input_vec1 = {
+5.4456432e-05
```

Comment:
In general looks good to me, but do we really need such a huge hardcoded vector? If so, maybe it makes sense to combine values by 8-10 elements per line. Or maybe we can replace these two vectors by single with 'sum' primitive results?